.github/workflows/compare-benchmarks.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: Benchmarks on AMD64
+permissions: read-all
+on: [pull_request]
+jobs:
+  benchmark-head:
+    uses: ./.github/workflows/benchmark-template.yaml
+    with:
+      ref: ${{ github.event.pull_request.head.sha }}
+
+  benchmark-base:
+    uses: ./.github/workflows/benchmark-template.yaml
+    with:
+      ref: ${{ github.event.pull_request.base.sha }}
```

Comment:
These two jobs run in parallel by default. Should we run them sequentially? Probably yes.

https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow

Reply:
Updated, it now runs sequentially.